extension/test/data_test.ts

```diff
@@ -69,4 +69,166 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    describe('with a word dict entry', function () {
+      // wordDictEntry was generated with: rcxDict.wordSearch('あ', /* doNames= */ false)
+      const wordDictEntry = {
+        kanji: '',
+        onkun: '',
+        nanori: '',
+        bushumei: '',
+        misc: {},
+        eigo: '',
+        hasNames: false,
+        data: [
+          { entry: 'あ /(int) (1) ah/oh/(int) (2) hey!/', reason: undefined },
```

Comment:
It's reasonable to use a real entry for this test data but my preference is to use fake data which draws the users focus to the parts of the input data which are critical.

In this case, the primary transformation is about taking the `/` separated parts and formatting them differently. In that case, you could do something like.

`entry: '<head1> /<sense 1.1>/<sense 1.2>/'`
`entry: '<head2> /<sense2.1>/<sense 2.2>/`

This really highlights that the actual text doesn't matter at all but the separator does. It also shows that the space in the head is removed in the formatted output. (I used `<>` since it's commonly used to show placeholders but something else (or nothing) might be more readable)

(of course, I didn't test this so if for example the `makeText` has some constraints on the input then we'd need to tweak the test data but that would again highlight exactly which parts of the input matter)

Reply:
Yeah I agree, using fake data that draws attention to what should be focused on will make the tests even easier to understand, so I'll do that.